week-2/Chen recipe-index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html> 
+<html>
+<head>
```

Comment:
Since the `<head>` and `<body>` are within `<html>`, they should be indented one step